code/datums/ruins/bridges/bridges.dm

```diff
@@ -109,6 +113,62 @@
 
 	T.flags |= LAVA_BRIDGE
 
+/obj/structure/bridge_walkway
+	name = "floor"
+	icon = 'icons/turf/floors.dmi'
+	layer = ABOVE_OPEN_TURF_LAYER
+	anchored = TRUE
+	resistance_flags = LAVA_PROOF | FIRE_PROOF | UNACIDABLE | ACID_PROOF
+	var/add_shadow = FALSE
+
+/obj/structure/bridge_walkway/Initialize(mapload, add_shadow_)
+	. = ..()
+	add_shadow = add_shadow_
+	return INITIALIZE_HINT_LATELOAD
+
+/obj/structure/bridge_walkway/LateInitialize()
+	. = ..()
+	var/turf/T = get_turf(src)
+	if(T && add_shadow)
+		T.layer = LATTICE_LAYER // prevent occlusion of drop shadows from bordering turfs
+		T.add_filter("AO", 1, drop_shadow_filter(x = 0, y = 0, size = 5, offset = 0, color = "#04080FEE"))
```

Comment:
Not sure the best way to handle this, but doesn't respect the `mymob.client.prefs.toggles & PREFTOGGLE_AMBIENT_OCCLUSION` pref

Reply:
I didn't realize how prominent the default AO for everything is, I'll just let that affect the appearance instead.